### ISSUE TITLE
How to use `ESLint Fixer` in `.vue:script`

### ISSUE THREAD

**Issue by ccforeverd** [Glavin001/atom-beautify#2026]
https://github.com/Glavin001/atom-beautify/blob/e0dc5c313ea0ae62c1de09724ce87769384af715/src/beautifiers/vue-beautifier.coffee#L31

I have my javascript files beautified with `atom-beautify` by engine `ESLint Fixer`, and also want to beautify the script tag in vue template files with `ESLint Fixer` to make the same javascript format.

How can I use? It's there some simple way?

Sorry to disturb you!

PS. I'm poor in English...

**Comment by szeck87** [comment]
Can you provide sample code of what it looks like preformatted and what you expect?

**Comment by range3** [comment]
Hi, I can use ESLint Fixer engine in my .vue files so far, but in an ad hoc way...

1. Install `eslint-plugin-vue`  to make .vue files eslintable
1. Configure atom-beautify.js.default_beautifier = "ESLint Fixer"
1. Open your .vue file and exec `atom-beautify:beautify-language-javascript` using Atom command palette

configure keymap.cson if you want
```
# beautify
'atom-text-editor':
  'ctrl-i': 'atom-beautify:beautify-editor'

# force use ESLint Fixer engine provided atom-beautify for js in .vue files
"atom-text-editor[data-grammar='text html vue']":
  'ctrl-i': 'atom-beautify:beautify-language-javascript'
```

**Comment by range3** [comment]
I created a pull request to add an option: 
```
"*":
  "atom-beautify":
    vue:
      default_beautifier: "ESLint Fixer"
```
Could you please review this PR ? 😊

**Comment by Glavin001** [comment]
Published to v0.32.0
